web/profiles/joinup/config/install/filter.format.basic_html.yml

```diff
@@ -6,8 +6,6 @@ dependencies:
 name: 'Basic HTML'
 format: basic_html
 weight: 0
-roles:
-  - authenticated
```

Comment:
Authenticated users are not allowed to use the Basic HTML filter format any more? This is unrelated to this ticket.

I'm guessing this has been exported like this, so this indicates a problem in our configuration. Still I would leave this unchanged. If this pops up again when re-exporting config then we can take a closer look.